Confluent Cloud initially has 12 months of cost history (#27692)
DataDog/documentation: content/en/cloud_cost_management/saas_costs.md

4. Under the **Resources** section, click the toggle for `Collect cost data to view in Cloud Cost Management`.
5. Click **Save**.

> Your Confluent Cloud cost data will be available in Cloud Cost Management 24 hours after setup. This data will automatically include 12 months of history, the maximum provided by the Confluent billing API. Over the next three months, the data will gradually expand to cover 15 months of history. To access the available data collected by each SaaS Cost Integration, see the [Data Collected section](#data-collected).

Reviewer: ```suggestion
Your Confluent Cloud cost data becomes available in Cloud Cost Management 24 hours after setup. This data automatically includes 12 months of history, the maximum provided by the Confluent billing API. Over the next three months, the data gradually expands to cover 15 months of history. To access the available data collected by each SaaS Cost Integration, see the [Data Collected section](#data-collected).
```
